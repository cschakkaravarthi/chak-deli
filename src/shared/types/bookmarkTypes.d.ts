import Bookmark from '../models/Bookmark.model';
export declare const LOAD_BOOKMARKS = "LOAD_BOOKMARKS";
export declare const FETCH_BOOKMARKS = "FETCH_BOOKMARKS";
export declare const SET_BOOKMARK = "SET_BOOKMARK";
export declare const SAVE_BOOKMARK = "SAVE_BOOKMARK";
export declare const DELETE_BOOKMARK = "DELETE_BOOKMARK";
export declare const REMOVE_BOOKMARK = "REMOVE_BOOKMARK";
export interface LoadBookmarksAction {
    type: typeof LOAD_BOOKMARKS;
    bookmarks: Bookmark[];
}
export interface SetBookmarkAction {
    type: typeof SET_BOOKMARK;
    success: boolean;
}
export interface DeleteBookmarkAction {
    type: typeof DELETE_BOOKMARK;
    success: boolean;
}
export interface BookmarkState {
    bookmarks: Bookmark[];
    success: boolean;
}
//# sourceMappingURL=bookmarkTypes.d.ts.map