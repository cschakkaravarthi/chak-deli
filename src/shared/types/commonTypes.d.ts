export declare const TOGGLE_GLOBAL_TOAST = "TOGGLE_GLOBAL_TOAST";
export interface ToggleGlobalToastAction {
    type: typeof TOGGLE_GLOBAL_TOAST;
    displayToast: boolean;
    toastMessage: string;
    toastError: boolean;
}
export interface CommonState {
    displayToast: boolean;
    toastMessage: string;
    toastError: boolean;
}
//# sourceMappingURL=commonTypes.d.ts.map