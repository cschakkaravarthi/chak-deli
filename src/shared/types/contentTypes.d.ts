import Link from '../models/Link.model';
import Article from '../models/Article.model';
import EventModel from '../models/Event.model';
import { Page } from '../models/Page.model';
import { DocumentSearch } from '../models/Document.model';
import { Department } from '../models/Department.model';
import BoxModel from '../models/Box.model';
export interface TaxonomyTermModel {
    drupal_id: number;
    title: string;
    weight?: number;
    icon?: any;
}
export interface BaseContentModel {
    type: string;
    drupal_id: number;
    title: string;
    summary?: string;
    categories: TaxonomyTermModel[];
    category_ancestors: TaxonomyTermModel[];
    owner: TaxonomyTermModel[];
}
export declare type VariousContentModel = Link | Article | EventModel | Page | DocumentSearch | Department | BoxModel;
export interface VariousContentGroupModel extends TaxonomyTermModel {
    content: VariousContentModel[];
}
export interface ContentStateModel {
    content: ContentCategorizedStorageModel;
    groupedContent: ContentGroupedCategorizedStorageModel;
}
export interface ContentGroupedCategorizedStorageModel {
    [key: string]: VariousContentGroupModel[];
}
export interface ContentCategorizedStorageModel {
    [key: string]: VariousContentModel[];
}
export declare const FETCH_CONTENT = "FETCH_CONTENT";
export declare const LOAD_CONTENT = "LOAD_CONTENT";
export declare const FETCH_GROUPED_CONTENT = "FETCH_GROUPED_CONTENT";
export declare const LOAD_GROUPED_CONTENT = "LOAD_GROUPED_CONTENT";
export interface LoadGroupedContentAction {
    type: typeof LOAD_GROUPED_CONTENT;
    content: VariousContentGroupModel[];
    filterId: string;
}
export interface LoadContentAction {
    type: typeof LOAD_CONTENT;
    content: VariousContentModel[];
    filterId: string;
}
export declare const dummyVariousContentGroup: VariousContentGroupModel;
export declare const dummyVariousContent: VariousContentModel[];
export declare const dummyContentGroupedCategorizedStorage: ContentGroupedCategorizedStorageModel;
//# sourceMappingURL=contentTypes.d.ts.map