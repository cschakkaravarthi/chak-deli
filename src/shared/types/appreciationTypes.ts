import { AppreciationCardNotificationModel as AlertModel } from '../models/Notification.model';
import { AppreciationCardTemplate, AppreciationCardsListGrouped } from '../models/AppreciationCards.model';

export const FETCH_APPRECIATION_DATA = 'FETCH_APPRECIATION_DATA';
export const LOAD_APPRECIATION_ALERT = 'LOAD_APPRECIATION_ALERT';
export const LOAD_APPRECIATION_ALERT_ERROR = 'LOAD_APPRECIATION_ALERT_ERROR';
export const CLEAR_APPRECIATION_REDUCER = 'CLEAR_APPRECIATION_REDUCER';
export const LOAD_APPRECIATION_MANAGER_ALERT = 'LOAD_APPRECIATION_MANAGER_ALERT';
export const SET_NEW_CARD_DETAIL = 'SET_NEW_CARD_DETAIL';
export const FETCH_CARD_TEMPLATES = 'FETCH_CARD_TEMPLATES';
export const CLEAR_NEW_CARD_FORM = 'CLEAR_NEW_CARD_FORM';
export const TOGGLE_SUCCESS_SENT_ALERT = 'TOGGLE_SUCCESS_SENT_ALERT';
export const GET_APPRECIATION_CARDS_LIST_GROUPED = 'GET_APPRECIATION_CARDS_LIST_GROUPED';
export const VIEW_APPRECIATION_CARD = 'VIEW_APPRECIATION_CARD';
export const APPRECIATION_CARD_ID = 'APPRECIATION_CARD_ID';
export const ARCHIVE_ALERT_STATE = 'ARCHIVE_ALERT_STATE';
export const LOAD_CARDS_IN_GROUP = 'LOAD_CARDS_IN_GROUP';

export interface AppreciationState {
  cardId: string;
  cardArchiveSuccess: boolean;
  isFetching: boolean;
  appreciationCardAlert?: AlertModel[];
  appreciationCardAlertError?: boolean;
  appreciationManagerCardAlert?: AlertModel[];
  newAppreciationCardForm: NewAppreciationCardForm;
  cardTemplates: AppreciationCardTemplate[];
  showSentSucessAlert: boolean;
  appreciationCardsListGrouped?: AppreciationCardsListGrouped;
  cardToView?: AlertModel;
  displayCardView: boolean;
}

export interface NewAppreciationCardForm {
  senderName?: string;
  templateId?: string;
  cardMessage?: string;
  recipientEmail?: string;
  recipientLastName?: string;
  recipientFirstName?: string;
  recipientSupervisorEmail?: string;
  managerName?: string;
}

export interface ArchiveAlertState {
  cardArchiveSuccess: boolean;
  type: typeof ARCHIVE_ALERT_STATE;
}

export interface SetArchiveAlert {
  cardId: string;
  type: typeof APPRECIATION_CARD_ID;
}

export interface FetchRequest {
  isFetching: boolean;
  type: typeof FETCH_APPRECIATION_DATA;
}

export interface LoadAppreciationAlertAction {
  isFetching: boolean;
  type: typeof LOAD_APPRECIATION_ALERT;
  appreciationCardAlert: AlertModel[];
}

export interface LoadAppreciationManagerAlertAction {
  isFetching: boolean;
  type: typeof LOAD_APPRECIATION_MANAGER_ALERT;
  appreciationManagerCardAlert: AlertModel[];
}

export interface AppreciationCardAlertError {
  isFetching: boolean;
  type: typeof LOAD_APPRECIATION_ALERT_ERROR;
  appreciationCardAlertError?: boolean;
}

export interface ClearAprreciationAction {
  type: typeof CLEAR_APPRECIATION_REDUCER;
}

export interface SetNewCardDetailAction {
  type: typeof SET_NEW_CARD_DETAIL;
  payload: NewAppreciationCardForm;
}

export interface SetDetailGroupAction {
  type: typeof SET_NEW_CARD_DETAIL;
  payload: NewAppreciationCardForm;
}

export interface FetchAppreciationCardTemplatesAction {
  type: typeof FETCH_CARD_TEMPLATES;
  cardTemplates: AppreciationCardTemplate[];
}

export interface ClearNewCardFormAction {
  type: typeof CLEAR_NEW_CARD_FORM;
}

export interface ToggleSentSuccessAlert {
  type: typeof TOGGLE_SUCCESS_SENT_ALERT;
  showSentSucessAlert: boolean;
}

export interface GetAppreciationCardsListGroupedAction {
  type: typeof GET_APPRECIATION_CARDS_LIST_GROUPED;
  appreciationCardsListGrouped: AppreciationCardsListGrouped;
}

export interface ViewAppreciationCardAction {
  type: typeof VIEW_APPRECIATION_CARD;
  displayCardView: boolean;
  cardToView?: AlertModel;
}

export interface AddAppreciationCardsToGroupAction {
  type: typeof LOAD_CARDS_IN_GROUP;
  appreciationCards: AlertModel[];
  cardsType: string;
}
