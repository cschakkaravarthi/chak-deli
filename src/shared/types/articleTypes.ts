import Article from '../models/Article.model';
import Facet, { FacetProperty } from '../models/Facet.model';
import { VariousContentModel } from './contentTypes';

export const LOAD_ARTICLES = 'LOAD_ARTICLES';
export const LOAD_ARTICLE_BY_ID = 'LOAD_ARTICLE_BY_ID';
export const FETCH_MORE_ARTICLES = 'LOAD_MORE_ARTICLES';
export const SET_ARTICLE_FILTER_FACET = 'SET_ARTICLE_FILTER_FACET';
export const CLEAR_ARTICLES_LIST = 'CLEAR_ARTICLES_LIST';
export const LIKE_ARTICLE_TOGGLE = 'LIKE_ARTICLE_TOGGLE';

export interface ArticleState {
  article?: Article;
  isLastPage: boolean;
  articleList: Article[];
  facets: Facet;
  selectedFacet: string;
  pageCount: number;
}

export interface LoadArticlesAction {
  type: typeof LOAD_ARTICLES;
  articleList: Article[];
}

export interface LoadArticleByIdAction {
  type: typeof LOAD_ARTICLE_BY_ID;
  article: VariousContentModel;
}
export interface LoadMoreArticlesAction {
  type: typeof FETCH_MORE_ARTICLES;
  articleList?: Article[];
  facets?: Facet;
}
export interface SetFilterFacetAction {
  type: typeof SET_ARTICLE_FILTER_FACET;
  selectedFacet: FacetProperty;
}

export interface ClearArticlesListAction {
  type: typeof CLEAR_ARTICLES_LIST;
}

export interface LikeArticleToggleAction {
  type: typeof LIKE_ARTICLE_TOGGLE;
  articleId: number;
  likedByUser: boolean;
}
