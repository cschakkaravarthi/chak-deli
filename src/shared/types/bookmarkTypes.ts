import Bookmark from '../models/Bookmark.model';

export const LOAD_BOOKMARKS = 'LOAD_BOOKMARKS';
export const FETCH_BOOKMARKS = 'FETCH_BOOKMARKS';
export const SET_BOOKMARK = 'SET_BOOKMARK';
export const SAVE_BOOKMARK = 'SAVE_BOOKMARK';
export const DELETE_BOOKMARK = 'DELETE_BOOKMARK';
export const REMOVE_BOOKMARK = 'REMOVE_BOOKMARK';

export interface LoadBookmarksAction {
  type: typeof LOAD_BOOKMARKS;
  bookmarks: Bookmark[];
}

export interface SetBookmarkAction {
  type: typeof SET_BOOKMARK;
  success: boolean;
}

export interface DeleteBookmarkAction {
  type: typeof DELETE_BOOKMARK;
  success: boolean;
}

export interface BookmarkState {
  bookmarks: Bookmark[];
  success: boolean;
}
