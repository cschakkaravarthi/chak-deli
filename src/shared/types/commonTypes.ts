import RejectModal from './../models/RejectModal';

export const CURRENT_USER_NAME = 'CURRENT_USER_NAME';
export const TOGGLE_GLOBAL_TOAST = 'TOGGLE_GLOBAL_TOAST';
export const SET_REJECT_MODAL_TEXTAREA = 'SET_REJECT_MODAL_TEXTAREA';
export const SET_REJECT_MODAL_ID_ISSHOW = 'SET_REJECT_MODAL_ID_ISSHOW';

export interface ToggleGlobalToastAction {
  type: typeof TOGGLE_GLOBAL_TOAST;
  displayToast: boolean;
  toastMessage: string;
  toastError: boolean;
}

export type CurrentUser = {
  family_name: string;
  given_name: string;
};

export interface SetCurrentUsername {
  type: typeof CURRENT_USER_NAME;
  currentUsername: CurrentUser;
}

export interface CommonState {
  currentUsername: CurrentUser;
  displayToast: boolean;
  toastMessage: string;
  toastError: boolean;
  idAndIsShow: RejectModal;
  textarea: string;
}

export interface SetRejectModalIdIsShow {
  type: typeof SET_REJECT_MODAL_ID_ISSHOW;
  idAndIsShow: RejectModal;
}

export interface SetRejectModalTextarea {
  type: typeof SET_REJECT_MODAL_TEXTAREA;
  textarea: string;
}
