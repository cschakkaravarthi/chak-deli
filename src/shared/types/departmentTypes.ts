import { Department } from '../models/Department.model';
import { FacetedContentModel } from './contentTypes';

export const LOAD_DEPARTMENT_BY_ID = 'LOAD_DEPARTMENT_BY_ID';
export const FETCH_DEPARTMENT_BY_ID = 'FETCH_DEPARTMENT_BY_ID';
export const LOAD_DEPARTMENTS = 'LOAD_DEPARTMENTS';
export const FETCH_DEPARTMENTS = 'FETCH_DEPARTMENTS';
export const FETCH_CONTACTS = 'FETCH_CONTACTS';

export interface DepartmentsState {
  departments: FacetedContentModel;
}

export interface LoadDepartmentByIdAction {
  type: typeof LOAD_DEPARTMENT_BY_ID;
  department: Department;
}

export interface LoadDepartmentsAction {
  type: typeof LOAD_DEPARTMENTS;
  departments: Department[];
}
