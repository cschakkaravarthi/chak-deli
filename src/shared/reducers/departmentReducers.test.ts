import departmentsReducers, { initialState } from './departmentReducers';
import { LOAD_DEPARTMENTS } from '../types/departmentTypes';
import { dummyDepartmentsArray } from '../models/Department.model';

describe('departmentsReducers', () => {
  it('Handles default case', () => {
    const state = departmentsReducers(undefined, {});
    expect(state).toEqual(initialState);
  });

  it('Handles loadDepartments action', () => {
    const action = {
      type: LOAD_DEPARTMENTS,
      departments: dummyDepartmentsArray
    };
    const state = departmentsReducers(initialState, action);
    expect(state).toMatchSnapshot();
  });
});
