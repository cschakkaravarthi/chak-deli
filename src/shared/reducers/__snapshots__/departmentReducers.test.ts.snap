// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`departmentsReducers Handles loadDepartments action 1`] = `
Object {
  "departments": Array [
    Object {
      "banner_alt": "Two people at a desk",
      "banner_uri": Object {
        "source": "s3://2019-11/helloquence-5fNmWej4tAA-unsplash.jpg",
        "umgc_banner": "s3://styles/umgc_banner/s3/2019-11/helloquence-5fNmWej4tAA-unsplash.jpg",
        "umgc_embedded": "s3://styles/umgc_embedded/s3/2019-11/helloquence-5fNmWej4tAA-unsplash.jpg",
        "umgc_featured": "s3://styles/umgc_featured/s3/2019-11/helloquence-5fNmWej4tAA-unsplash.jpg",
        "umgc_thumbnail": "s3://styles/umgc_thumbnail/s3/2019-11/helloquence-5fNmWej4tAA-unsplash.jpg",
      },
      "body": "<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Praesent vitae odio vehicula nisl lacinia luctus non et libero. Quisque leo tortor, lacinia et porttitor vel, dignissim at sapien.</p>",
      "categories": Array [
        Object {
          "drupal_id": 23,
          "icon": "",
          "title": "Departments List",
          "weight": 2,
        },
      ],
      "category_ancestors": Array [
        Object {
          "drupal_id": 23,
          "icon": "",
          "title": "Departments List",
          "weight": 2,
        },
      ],
      "contacts": Array [
        Object {
          "contacts": Array [
            "test1@example.com",
            "test2@example.com",
          ],
          "group_label": "Primary Contacts",
        },
        Object {
          "contacts": Array [
            "test3@example.com",
            "test4@example.com",
          ],
          "group_label": "Secondary Contacts",
        },
      ],
      "drupal_id": 192019,
      "is_homepage": false,
      "language": "en",
      "logo_uri": "s3://2019-11/helloquence-5fNmWej4tAA-unsplash.jpg",
      "owner": Array [],
      "score": undefined,
      "summary": "body Lorem ipsum dolor summary",
      "template": Array [
        Object {
          "drupal_id": 23,
          "icon": "",
          "title": "Template A",
          "weight": 2,
        },
      ],
      "title": "Finance",
      "type": "department",
      "website_url": "",
    },
    Object {
      "banner_alt": "Two people at a desk",
      "banner_uri": Object {
        "source": "s3://2019-11/helloquence-5fNmWej4tAA-unsplash.jpg",
        "umgc_banner": "s3://styles/umgc_banner/s3/2019-11/helloquence-5fNmWej4tAA-unsplash.jpg",
        "umgc_embedded": "s3://styles/umgc_embedded/s3/2019-11/helloquence-5fNmWej4tAA-unsplash.jpg",
        "umgc_featured": "s3://styles/umgc_featured/s3/2019-11/helloquence-5fNmWej4tAA-unsplash.jpg",
        "umgc_thumbnail": "s3://styles/umgc_thumbnail/s3/2019-11/helloquence-5fNmWej4tAA-unsplash.jpg",
      },
      "body": "<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Praesent vitae odio vehicula nisl lacinia luctus non et libero. Quisque leo tortor, lacinia et porttitor vel, dignissim at sapien.</p>",
      "categories": Array [
        Object {
          "drupal_id": 23,
          "icon": "",
          "title": "Departments List",
          "weight": 2,
        },
      ],
      "category_ancestors": Array [
        Object {
          "drupal_id": 23,
          "icon": "",
          "title": "Departments List",
          "weight": 2,
        },
      ],
      "contacts": Array [
        Object {
          "contacts": Array [
            "test1@example.com",
            "test2@example.com",
          ],
          "group_label": "Primary Contacts",
        },
        Object {
          "contacts": Array [
            "test3@example.com",
            "test4@example.com",
          ],
          "group_label": "Secondary Contacts",
        },
      ],
      "drupal_id": 1,
      "is_homepage": false,
      "language": "en",
      "logo_uri": "s3://2019-11/helloquence-5fNmWej4tAA-unsplash.jpg",
      "owner": Array [],
      "score": undefined,
      "summary": "body Lorem ipsum dolor summary",
      "template": Array [
        Object {
          "drupal_id": 23,
          "icon": "",
          "title": "Template A",
          "weight": 2,
        },
      ],
      "title": "Finance",
      "type": "department",
      "website_url": "",
    },
    Object {
      "banner_alt": "Two people at a desk",
      "banner_uri": Object {
        "source": "s3://2019-11/helloquence-5fNmWej4tAA-unsplash.jpg",
        "umgc_banner": "s3://styles/umgc_banner/s3/2019-11/helloquence-5fNmWej4tAA-unsplash.jpg",
        "umgc_embedded": "s3://styles/umgc_embedded/s3/2019-11/helloquence-5fNmWej4tAA-unsplash.jpg",
        "umgc_featured": "s3://styles/umgc_featured/s3/2019-11/helloquence-5fNmWej4tAA-unsplash.jpg",
        "umgc_thumbnail": "s3://styles/umgc_thumbnail/s3/2019-11/helloquence-5fNmWej4tAA-unsplash.jpg",
      },
      "body": "<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Praesent vitae odio vehicula nisl lacinia luctus non et libero. Quisque leo tortor, lacinia et porttitor vel, dignissim at sapien.</p>",
      "categories": Array [
        Object {
          "drupal_id": 23,
          "icon": "",
          "title": "Departments List",
          "weight": 2,
        },
      ],
      "category_ancestors": Array [
        Object {
          "drupal_id": 23,
          "icon": "",
          "title": "Departments List",
          "weight": 2,
        },
      ],
      "contacts": Array [
        Object {
          "contacts": Array [
            "test1@example.com",
            "test2@example.com",
          ],
          "group_label": "Primary Contacts",
        },
        Object {
          "contacts": Array [
            "test3@example.com",
            "test4@example.com",
          ],
          "group_label": "Secondary Contacts",
        },
      ],
      "drupal_id": 2,
      "is_homepage": false,
      "language": "en",
      "logo_uri": "s3://2019-11/helloquence-5fNmWej4tAA-unsplash.jpg",
      "owner": Array [],
      "score": undefined,
      "summary": "body Lorem ipsum dolor summary",
      "template": Array [
        Object {
          "drupal_id": 23,
          "icon": "",
          "title": "Template A",
          "weight": 2,
        },
      ],
      "title": "Finance",
      "type": "department",
      "website_url": "",
    },
    Object {
      "banner_alt": "Two people at a desk",
      "banner_uri": Object {
        "source": "s3://2019-11/helloquence-5fNmWej4tAA-unsplash.jpg",
        "umgc_banner": "s3://styles/umgc_banner/s3/2019-11/helloquence-5fNmWej4tAA-unsplash.jpg",
        "umgc_embedded": "s3://styles/umgc_embedded/s3/2019-11/helloquence-5fNmWej4tAA-unsplash.jpg",
        "umgc_featured": "s3://styles/umgc_featured/s3/2019-11/helloquence-5fNmWej4tAA-unsplash.jpg",
        "umgc_thumbnail": "s3://styles/umgc_thumbnail/s3/2019-11/helloquence-5fNmWej4tAA-unsplash.jpg",
      },
      "body": "<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Praesent vitae odio vehicula nisl lacinia luctus non et libero. Quisque leo tortor, lacinia et porttitor vel, dignissim at sapien.</p>",
      "categories": Array [
        Object {
          "drupal_id": 23,
          "icon": "",
          "title": "Departments List",
          "weight": 2,
        },
      ],
      "category_ancestors": Array [
        Object {
          "drupal_id": 23,
          "icon": "",
          "title": "Departments List",
          "weight": 2,
        },
      ],
      "contacts": Array [
        Object {
          "contacts": Array [
            "test1@example.com",
            "test2@example.com",
          ],
          "group_label": "Primary Contacts",
        },
        Object {
          "contacts": Array [
            "test3@example.com",
            "test4@example.com",
          ],
          "group_label": "Secondary Contacts",
        },
      ],
      "drupal_id": 3,
      "is_homepage": false,
      "language": "en",
      "logo_uri": "s3://2019-11/helloquence-5fNmWej4tAA-unsplash.jpg",
      "owner": Array [],
      "score": undefined,
      "summary": "body Lorem ipsum dolor summary",
      "template": Array [
        Object {
          "drupal_id": 23,
          "icon": "",
          "title": "Template A",
          "weight": 2,
        },
      ],
      "title": "Finance",
      "type": "department",
      "website_url": "",
    },
    Object {
      "banner_alt": "Two people at a desk",
      "banner_uri": Object {
        "source": "s3://2019-11/helloquence-5fNmWej4tAA-unsplash.jpg",
        "umgc_banner": "s3://styles/umgc_banner/s3/2019-11/helloquence-5fNmWej4tAA-unsplash.jpg",
        "umgc_embedded": "s3://styles/umgc_embedded/s3/2019-11/helloquence-5fNmWej4tAA-unsplash.jpg",
        "umgc_featured": "s3://styles/umgc_featured/s3/2019-11/helloquence-5fNmWej4tAA-unsplash.jpg",
        "umgc_thumbnail": "s3://styles/umgc_thumbnail/s3/2019-11/helloquence-5fNmWej4tAA-unsplash.jpg",
      },
      "body": "<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Praesent vitae odio vehicula nisl lacinia luctus non et libero. Quisque leo tortor, lacinia et porttitor vel, dignissim at sapien.</p>",
      "categories": Array [
        Object {
          "drupal_id": 23,
          "icon": "",
          "title": "Departments List",
          "weight": 2,
        },
      ],
      "category_ancestors": Array [
        Object {
          "drupal_id": 23,
          "icon": "",
          "title": "Departments List",
          "weight": 2,
        },
      ],
      "contacts": Array [
        Object {
          "contacts": Array [
            "test1@example.com",
            "test2@example.com",
          ],
          "group_label": "Primary Contacts",
        },
        Object {
          "contacts": Array [
            "test3@example.com",
            "test4@example.com",
          ],
          "group_label": "Secondary Contacts",
        },
      ],
      "drupal_id": 5,
      "is_homepage": false,
      "language": "en",
      "logo_uri": "s3://2019-11/helloquence-5fNmWej4tAA-unsplash.jpg",
      "owner": Array [],
      "score": undefined,
      "summary": "body Lorem ipsum dolor summary",
      "template": Array [
        Object {
          "drupal_id": 23,
          "icon": "",
          "title": "Template A",
          "weight": 2,
        },
      ],
      "title": "Finance",
      "type": "department",
      "website_url": "",
    },
    Object {
      "banner_alt": "Two people at a desk",
      "banner_uri": Object {
        "source": "s3://2019-11/helloquence-5fNmWej4tAA-unsplash.jpg",
        "umgc_banner": "s3://styles/umgc_banner/s3/2019-11/helloquence-5fNmWej4tAA-unsplash.jpg",
        "umgc_embedded": "s3://styles/umgc_embedded/s3/2019-11/helloquence-5fNmWej4tAA-unsplash.jpg",
        "umgc_featured": "s3://styles/umgc_featured/s3/2019-11/helloquence-5fNmWej4tAA-unsplash.jpg",
        "umgc_thumbnail": "s3://styles/umgc_thumbnail/s3/2019-11/helloquence-5fNmWej4tAA-unsplash.jpg",
      },
      "body": "<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Praesent vitae odio vehicula nisl lacinia luctus non et libero. Quisque leo tortor, lacinia et porttitor vel, dignissim at sapien.</p>",
      "categories": Array [
        Object {
          "drupal_id": 23,
          "icon": "",
          "title": "Departments List",
          "weight": 2,
        },
      ],
      "category_ancestors": Array [
        Object {
          "drupal_id": 23,
          "icon": "",
          "title": "Departments List",
          "weight": 2,
        },
      ],
      "contacts": Array [
        Object {
          "contacts": Array [
            "test1@example.com",
            "test2@example.com",
          ],
          "group_label": "Primary Contacts",
        },
        Object {
          "contacts": Array [
            "test3@example.com",
            "test4@example.com",
          ],
          "group_label": "Secondary Contacts",
        },
      ],
      "drupal_id": 6,
      "is_homepage": false,
      "language": "en",
      "logo_uri": "s3://2019-11/helloquence-5fNmWej4tAA-unsplash.jpg",
      "owner": Array [],
      "score": undefined,
      "summary": "body Lorem ipsum dolor summary",
      "template": Array [
        Object {
          "drupal_id": 23,
          "icon": "",
          "title": "Template A",
          "weight": 2,
        },
      ],
      "title": "Finance",
      "type": "department",
      "website_url": "",
    },
    Object {
      "banner_alt": "Two people at a desk",
      "banner_uri": Object {
        "source": "s3://2019-11/helloquence-5fNmWej4tAA-unsplash.jpg",
        "umgc_banner": "s3://styles/umgc_banner/s3/2019-11/helloquence-5fNmWej4tAA-unsplash.jpg",
        "umgc_embedded": "s3://styles/umgc_embedded/s3/2019-11/helloquence-5fNmWej4tAA-unsplash.jpg",
        "umgc_featured": "s3://styles/umgc_featured/s3/2019-11/helloquence-5fNmWej4tAA-unsplash.jpg",
        "umgc_thumbnail": "s3://styles/umgc_thumbnail/s3/2019-11/helloquence-5fNmWej4tAA-unsplash.jpg",
      },
      "body": "<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Praesent vitae odio vehicula nisl lacinia luctus non et libero. Quisque leo tortor, lacinia et porttitor vel, dignissim at sapien.</p>",
      "categories": Array [
        Object {
          "drupal_id": 23,
          "icon": "",
          "title": "Departments List",
          "weight": 2,
        },
      ],
      "category_ancestors": Array [
        Object {
          "drupal_id": 23,
          "icon": "",
          "title": "Departments List",
          "weight": 2,
        },
      ],
      "contacts": Array [
        Object {
          "contacts": Array [
            "test1@example.com",
            "test2@example.com",
          ],
          "group_label": "Primary Contacts",
        },
        Object {
          "contacts": Array [
            "test3@example.com",
            "test4@example.com",
          ],
          "group_label": "Secondary Contacts",
        },
      ],
      "drupal_id": 7,
      "is_homepage": false,
      "language": "en",
      "logo_uri": "s3://2019-11/helloquence-5fNmWej4tAA-unsplash.jpg",
      "owner": Array [],
      "score": undefined,
      "summary": "body Lorem ipsum dolor summary",
      "template": Array [
        Object {
          "drupal_id": 23,
          "icon": "",
          "title": "Template A",
          "weight": 2,
        },
      ],
      "title": "Finance",
      "type": "department",
      "website_url": "",
    },
  ],
}
`;
